packages/js-dapi-client/lib/test/bootstrap.js

```diff
@@ -20,4 +20,16 @@ afterEach(function afterEach() {
   this.sinon.restore();
 });
 
+before(function before() {
```

Comment:
You can keep only `this.sinon.restore();` in beforeEach

Reply:
Fixed